api/client/src/main/java/org/projectnessie/client/http/impl/jdk11/JavaHttpClient.java

```diff
@@ -65,6 +65,9 @@ final class JavaHttpClient implements org.projectnessie.client.http.HttpClient {
 
   @Override
   public HttpRequest newRequest(URI baseUri) {
+    if (client == null) {
+      throw new IllegalStateException("Client already closed");
```

Comment:
Could you reference #8316 in the description?

Reply:
Not sure whether this and 8316 are the same?

Reply:
ok